pkg/ebpf/tracer_windows.go

```diff
@@ -83,39 +130,27 @@ func (t *Tracer) expvarStats() {
 	}
 }
 
-// Stop function stops running tracer
-func (t *Tracer) Stop() {}
-
-func openDriverFile(path string) (windows.Handle, error) {
-	p, err := windows.UTF16PtrFromString(path)
-	if err != nil {
-		return windows.InvalidHandle, err
-	}
-	log.Debug("Creating Driver handle...")
-	h, err := windows.CreateFile(p,
-		windows.GENERIC_READ|windows.GENERIC_WRITE,
-		windows.FILE_SHARE_READ|windows.FILE_SHARE_WRITE,
-		nil,
-		windows.OPEN_EXISTING,
-		windows.FILE_FLAG_OVERLAPPED,
-		windows.Handle(0))
-	if err != nil {
-		return windows.InvalidHandle, err
-	}
-	log.Info("Connected to driver and handle created")
-	return h, nil
-}
-
-func closeDriverFile(handle windows.Handle) error {
-	return windows.CloseHandle(handle)
-}
+func (t *Tracer) initPacketPolling() (err error) {
+	log.Debugf("Started packet polling")
+	go func() {
```

Comment:
Do we need a clean way to terminate this loop?

Reply:
I think it would be good to do, but I wasn't sure about the best way of ensuring that. Would it be best to have a channel indicating exit for this, or is there another scheme to do this?

Reply:
A channel would be a good way to do this. You can use `select` to wait until the channel is closed  and then exit the loop. Something like:



```
func initPacketPolling(exit <-chan struct{})

for {
  select {
     case _, ok <- exit:
           return

     default:
         err = windows.GetQueued(....)
  }
}
```

Reply:
Thanks for the note, I've added this functionality!